inlong-sort-standalone/sort-standalone-source/src/main/java/org/apache/inlong/sort/standalone/sink/cls/ClsSinkContext.java

```diff
@@ -280,9 +293,9 @@ public void addSendResultMetric(ProfileEvent currentRecord, String bid, boolean
     /**
      * Get report dimensions.
      *
-     * @param currentRecord Event.
-     * @param bid Topic or dest ip.
-     * @return Prepared dimensions map.
+     * @param  currentRecord Event.
+     * @param  bid           Topic or dest ip.
```

Comment:
It is not necessary to indent the method parameters.
Please refer to this code-style link and reformat again, thanks.

https://github.com/apache/incubator-inlong/discussions/3082